api/migrations/versions/0237_update_sms_rates.py

```diff
@@ -0,0 +1,31 @@
+"""
+
+Revision ID: 0237
+Revises: 0236
+Create Date: 2020-03-18 22:13:57.400373
+
+"""
+import uuid
+
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = '0237'
+down_revision = '0236'
+
+
+def upgrade():
+    op.get_bind()
+    op.execute("UPDATE provider_rates SET rate = 6.0 WHERE provider_id = (SELECT id FROM provider_details WHERE identifier = 'telstra')")
+    op.execute("UPDATE provider_rates SET rate = 6.0 WHERE provider_id = (SELECT id FROM provider_details WHERE identifier = 'twilio')")
+    op.execute("UPDATE rates SET rate = 0.06 WHERE valid_from = '2017-06-30 14:00:00'")
+    op.execute("UPDATE rates SET rate = 0.06 WHERE valid_from = '2016-05-18 00:00:00'")
+
+
+def downgrade():
+    op.get_bind()
+    op.execute("UPDATE provider_rates SET rate = 1.8 WHERE provider_id = (SELECT id FROM provider_details WHERE identifier = 'telstra')")
```

Comment:
@acodegirl it doesn't really matter for these downgrade steps, but these 4 SQL commands should be run in reverse order from those in the upgrade steps.

Reply:
Addressed. 
I noticed in many other scripts we are executing downgrade's SQL commands in the same order as upgrade.

Reply:
@acodegirl yeah it's all a bit messy really. it's probably just good practice so thought i would mention it :)

Reply:
@jonathaningram It's a good learning for me. thanks for letting me know.  :)